downloads/swan-lake-release-notes/swan-lake-beta6/RELEASE_NOTE.md

```diff
@@ -0,0 +1,186 @@
+
+### Improvements
+
+#### `http` Package
+Allowed listener-level interceptors to have only the default path.
+
+### Breaking Changes
+
+#### `http` Package
+- Changed the `RequestContext:add` function to `RequestContext:set`.
+
+  ```ballerina
+  service class DefaultRequestInterceptor {
+      *http:RequestInterceptor;
+
+      resource function 'default [string... path](http:RequestContext ctx) returns http:NextService|error? {
+          ctx.set("last-interceptor", "default-interceptor");
+          return ctx.next();
+      }
+  }
+  ```
+- Changed the `parseHeader()` function to support multiple header values.
+
+  ```ballerina
+  http:HeaderValue[] values = check http:parseHeader("text/plain;level=1;q=0.6, application/xml;level=2");
+  ```
+
+### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Beta6](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aclosed+is%3Aissue+milestone%3A%22Swan+Lake+Beta6%22+label%3AType%2FBug).
+
+## Developer Tools Updates
+
+### Improvements
+
+#### OpenAPI Tool 
+
+- Added the `--with-tests` option for the OpenAPI client generation command, which will generate a test file template for all the relevant remote functions of the client. For example,
+
+  > `bal openapi -i <openapi contract> --mode client --with-tests`
+
+- Added the `--export-openapi` option for the `build` command, which will generate OpenAPI contract files for all the services in the current package. For example,
+
+  > `bal build --export-openapi`
+
+#### CLI
+
+- Introduced the `--target-dir` flag to the `run`, `test`, `doc`, and `clean` commands. With this, you can pass a custom directory to the aforementioned commands.
+- Added support for creating packages using template packages in Ballerina Central. For example, `bal new -t wso2/choreo_sample`.
+
+### Bug Fixes
+
+To view bug fixes, see the GitHub milestone for Swan Lake Beta3 of the repositories below.
+
+- [Language Server](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Beta6%22+label%3AType%2FBug+label%3ATeam%2FLanguageServer)
+- [OpenAPI](https://github.com/ballerina-platform/ballerina-openapi/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Beta6%22+label%3AType%2FBug)
+- [Debugger](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+label%3AType%2FBug+label%3AArea%2FDebugger+milestone%3A%22Ballerina+Swan+Lake+-+Beta6%22+is%3Aclosed)
+
+## Ballerina Packages
+
+### New Features
+
+Added support to add an `icon` field under the `[package]` table. For example,
+```toml
+[package]
+org = "ballerinax"
+name = "googleapis.gmail"
+version = "0.1.0”
+icon = “images/icon.png”
+```
+
+>**Info:** The specified icon file will be packaged into the `docs/` directory of the Ballerina archive (`.bala`). Only the `.png` format is supported for the icon.
+
+### Breaking Changes
+
+#### Test Framework 
```

Comment:
Why `Test Framework` is listed under `Ballerina Packages`? Shouldn't we have it under `Developer Tools Updates` @azinneera

Reply:
Fixed.